internal/pgbackrest/reconcile_test.go

```diff
@@ -133,7 +134,19 @@ func TestAddRepoVolumesToPod(t *testing.T) {
 						}
 					}
 					if !foundVolumeMount {
-						t.Error(fmt.Errorf("volume mount %s is missing or invalid", r.Name))
+						t.Error(fmt.Errorf("container volume mount %s is missing or invalid", r.Name))
+					}
+				}
+				for _, c := range template.Spec.InitContainers {
+					var foundVolumeMount bool
+					for _, vm := range c.VolumeMounts {
+						if vm.Name == r.Name && vm.MountPath == "/pgbackrest/"+r.Name {
+							foundVolumeMount = true
+							break
+						}
+					}
+					if !foundVolumeMount {
+						t.Error(fmt.Errorf("init container volume mount %s is missing or invalid", r.Name))
```

Comment:
⛏️ These don't need to be `error` objects.
```suggestion
						t.Errorf("init container volume mount %s is missing or invalid", r.Name)
```

Reply:
Updated.